Opserver/Views/Dashboard/Dashboard.Table.cshtml

```diff
@@ -87,7 +88,7 @@ else
                                     @if (n.MemoryUsed >= 0)
                                     {
                                         <div class="progress" style="width: 215px;">
-                                            <div class="progress-bar @n.MemoryMonitorStatus().ProgressBarClass()" style="width: @(n.PercentMemoryUsed.ToString())%"></div>
+                                            <div class="progress-bar @n.MemoryMonitorStatus().ProgressBarClass()" style="width: @(n.PercentMemoryUsed.Value.ToString(CultureInfo.InvariantCulture))%"></div>
```

Comment:
Yes I see that. I assume that the previous `@if (n.MemoryUsed >= 0)` was enough.